file/config.go

```diff
@@ -10,10 +12,10 @@ type Config struct {
 	Suffix string
 
 	// session value serialize func
-	SerializeFunc func(data map[string]interface{}) ([]byte, error)
+	SerializeFunc func(data *dictpool.Dict) ([]byte, error)
```

Comment:
that just broke backward compatibility, am I right?

Reply:
Yes, it broke backward compatibility.
I prefer to use dictpool, that it use slice store, instead of map that it's more slower.

Reply:
The all next changes, i will try to not break anything else.

This changes, i think was necessary because ccmap are very complex and slower.